packages/core/src/modules/credentials/protocol/v1/V1CredentialService.ts

```diff
@@ -125,10 +125,10 @@ export class V1CredentialService extends CredentialService {
     }
 
     const config: CredentialProposeOptions = {
```

Comment:
why are we calling the format service here? We don't use the attachment, is it for the linked attachments? I'm not sure

Reply:
Yes it looks like the createProposal method in the format service adds the linked attachments to the proposal but it also validates the proposal payload...I assume this is required as it relates to the interoperability issue we hit last week.

Reply:
Ok, could we maybe add a comment explaining this? Future readers of the code can maybe be confused that we aren't using the result